shared/AppInsightsCore/src/JavaScriptSDK/HelperFuncs.ts

```diff
@@ -310,21 +311,22 @@ export function strContains(value: string, search: string) {
  * Check if an object is of type Date
  */
 export function isDate(obj: any): obj is Date {
-    return !!(obj && objToString(obj) === "[object Date]");
+    return !!(obj && _objToString.call(obj) === "[object Date]");
 }
 
 /**
  * Check if an object is of type Array
  */
-export function isArray<T = any>(obj: any): obj is Array<T> {
-    return _isArray ? _isArray(obj) : !!(obj && objToString(obj) === "[object Array]");
+export let isArray: <T = any>(obj: any) => obj is Array<T> = _isArray || _isArrayPoly;
```

Comment:
Perf improvement to avoid an additional function call.

so now `isArray` will either be 
- an alias to `Array.isArray` when supported (So becomes a direct native call)
- an alias to the previous polyfill function when not. (stays as the previous poly function)